router/v3/channels.go

```diff
@@ -28,27 +30,34 @@ func (h *Handlers) GetChannels(c echo.Context) error {
 		return herror.BadRequest("include-dm and path cannot be specified at the same time")
 	}
 
-	res := echo.Map{
-		"public": h.ChannelManager.PublicChannelTree(),
-	}
+	var res echo.Map
 
 	if len(c.QueryParam("path")) > 0 {
 		channelPath := c.QueryParam("path")
```

Comment:
これは好みの問題な気がしますが、以下のようにするとより見やすそう？
```suggestion
	if channelPath := c.QueryParam("path"); channelPath != '' {
```

Reply:
確かに!
goの記法すっかり忘れてました